app/views/book_searches/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to "New search", books_path %>
+<h3> <%= t('book_searches.index.header', param: params[:search]) %> </h3>
+
+<ul class="search_results">
+  <% @book_results.each do |book| %>
+    <li>
```

Comment:
If this is the same as the item rendered in the books index then you may want to extract a partial for it